ramls/loan.json

```diff
@@ -253,6 +253,11 @@
         }
       }
     },
+    "feefine": {
```

Comment:
What does this represent, is it the total outstanding amount of all fees and fines for this loan?

Reply:
Are you referring to the description of the field or the implementation?

Reply:
The description has been changed in the latest commit